### DISCUSSION TITLE
Security Onion Installation Hangs - Security Onion-2.4.60-20240320 ISO

### DISCUSSION THREAD

**Discussion by catalinchertes** [Security-Onion-Solutions/securityonion#12866] [Unsupported Versions] [answered]
Hello,

I'm trying to install Security Onion on a VM with the official ISO in Proxmox on a host with this specs:

CPU: 12 Cores
RAM: 32 GB RAM
STORAGE: 500GB HDD

The problem occurs after the graphical installation finishes, in the installation script at this stage:

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/57810465/e920fba5-a0a4-42fc-abf0-d8e5bdf353b1)

It just hangs here forever, waited over 3 hours on this process but it never finishes or updates.

What could be the issue?

Thanks.

**Comment by cm-ops** [comment]
Do you see any issues in the `/root/sosetup.log`?

**Reply by catalinchertes** [reply]
No issues seen in /root/sosetup.log just the same message stuck on "Executing command '/usr/sbin/so-rule-update' in directory '/root'

Yesterday I canceled the process with CTRL + C and ran the script manually as root, it took around 2 hours and it completed successfully. 

How long does the installation process usually take? Today I'm running the process again as I had to reinstall the VM and it's stuck again at this point, running for more than 3 hours now.

**Comment by catalinchertes** [comment]
After digging a little bit I found out that what actually happens is that it fails to download the suricata rules when it runs the so-rule-update command, it gets stuck at 73%. I tried to download them manually with wget and it works with no problems, but the script dies trying to download this rules...

Tried using google DNS, internal DNS, removing the sniffing interface but nothing works.

If I run the so-rule-update manually, it gets stuck as well downloading this file, it starts and just stops randomly at 72-73%...

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/57810465/933f8478-6b7b-465f-8ea7-cfca5949780a)

**Comment by catalinchertes** [comment, marked as answer]
I finally ended up going for the airgap installation and everything works perfectly.

Thanks.
